tooling-e2e-tests/scheduler_scenario.sh

```
@@ -0,0 +1,114 @@
+
+if [[ -n "${TESTNET:-}" ]]; then
+  source "${SCRIPT_DIR}/testnet_env.sh"
+else
+  source "${SCRIPT_DIR}/local_env.sh"
+fi
+source "${SCRIPT_DIR}/utils.sh"
+
+# ${1} - actor name
+# ${2} - actor private key
+scenario() {
+  configure_cli ${1} ${2}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)